app/controllers/report_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ReportController < ApplicationController
+  def month_report
+    @current_user = current_user
+    @transactions = FinancesUser.includes(:finance).by_user_id(@current_user.id)
+
+    income_transactions = @transactions.by_status(true)
+    expense_transactions = @transactions - income_transactions
+    gon.hash_gon = { income_amout: income_transactions.pluck(:amout).sum,
```

Comment:
```suggestion
    gon.hash_gon = {income_amout: income_transactions.pluck(:amout).sum,
```
trước } và sau { k để spaces nhé

Reply:
vâng ạ.